client/stylesheets/_modal.scss

```diff
@@ -670,9 +674,11 @@
 .modAddListingTrigger {
 	background-color: rgba(255, 255, 255, 0) !important;
 	color: #f8504b !important;
-	font-size: 0.9rem !important;
+	font-size: 0.8rem !important;
 	padding: 18px 50px 18px 0px !important;
-	width: 100% !important; 
+	width: 100% !important;
+	margin-left: -30px !important;
+	font-family: 'Droid Sans', sans-serif !important; 
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Line contains trailing whitespace